ui/lib/config-ui/addon/components/login-settings/page/details.hbs

```diff
@@ -0,0 +1,67 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+{{#if @rule}}
+  <PageHeader as |p|>
+    <p.top>
+      <Page::Breadcrumbs @breadcrumbs={{@breadcrumbs}} />
+    </p.top>
+    <p.levelLeft>
+      <h1 class="title is-3">
+        {{@rule.name}}
+      </h1>
+    </p.levelLeft>
+  </PageHeader>
+
+  <nav class="tabs" aria-label="navigation for rule details">
+    <ul>
+      <li>
+        <LinkTo @route="login-settings.rule.details" @model={{@rule}} data-test-rule-tab>
+          Details
+        </LinkTo>
+      </li>
+    </ul>
+  </nav>
+
+  <Toolbar>
+    <ToolbarActions>
+      {{#if (has-capability this.capabilities "delete" id=@rule.name)}}
+        <Hds::Button
+          @text="Delete Rule"
+          @color="secondary"
+          class="toolbar-button"
+          {{on "click" (fn (mut this.showConfirmModal) true)}}
+          data-test-rule-delete
```

Comment:
Coming soon, a generic button selector for actions. Your safe at the moment because it doesn't yet exist.